testkit/src/lib.rs

```diff
@@ -242,6 +243,19 @@ impl TestKit {
             .create()
     }
 
+    /// Returns the list of the Instances enabled by default.
+    ///
+    /// Currently only `Supervisor` service is included.
+    fn default_service_instances() -> Vec<InstanceCollection> {
```

Comment:
In my PR with Configure interface I refuse of default service instances in Testkit. I think that you should completely remove the supervisor dependency in the testkit.

Reply:
Currently it's not really possible.
Let's wait until your PR will be merged then.